docs/release/release_0_4_11.md

```diff
@@ -0,0 +1,219 @@
+# napari 0.4.11
+
+We're happy to announce the release of napari 0.4.11!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+
+
+## New Features
+- Add context menu on layer list, introduce `QtActionContextMenu`. (#2556)
+- Add activity dialog and style progress bars (#2656)
+- Add playback options to settings (#2933)
+- Add autoscale modes to image layer model, and buttons to GUI (#3022)
+- Arbitrary plane rendering prototype (#3023)
+- Add view ray and labels selection in 3D (#3037)
+- Grid mode popup (#3084)
+- Add 3D fill, "mill", and "print" on top of #3074 (#3108)
+- Add positive tail length to tracks layer (#3138)
+- Arbitrary clipping planes for volumes in the image layer (#3140)
+- Mask image from points layer (#3151)
+
+
+## Improvements
+- Add `assign_[plugin]_to_extension` methods on plugin_manager.  (#2695)
+- Use QDoubleRangeSlider from superqt package (#2752)
+- Use labeled sliders from superqt (#2753)
+- Shortcuts UI (#2864)
+- Convert TextManager to an EventedModel (#2885)
+- Make maximum brush size flexible (#2901)
+- Allow layer to register action on double clicks. (#2907)
+- Reduce numpy array traceback print (#2910)
+- Provide manual deepcopy implementation for translations strings. (#2913)
+- Make Points construction with properties consistent with setting properties (#2916)
+- Add search field to plugin dialog  (#2923)
+- Refactor settings manager to allow setting all preferences with env vars and context managers (#2932)
```

Comment:
might want to mention this... though it's mostly a refactor, the "feature" is the ability to now set anything with env vars

Reply:
moved to feature